### DISCUSSION TITLE
Ability to Share Notebooks & Topics

### DISCUSSION THREAD

**Discussion by Bizzet** [streetwriters/notesnook#1952] [Q&A]
I think it would be a great addition if you allowed users to share entire notebooks and topics with others, similar to how the Monographs work.

I would personally like to see the abilities to:

- Make a private link for users to browse through entire notebooks using a web version if they have the link

> (OneNote does a great job at this. Here’s some information regarding their sharing feature: 
> https://support.microsoft.com/en-us/office/share-a-page-of-notes-or-an-entire-notebook-from-onenote-for-windows-10-d4a74a14-44a3-411e-8fb5-06e73ddf047f)

- Make a private link for users to browse through entire topics using a web version if they have the link

- Share notebooks and topics with other Notesnook users so they can download it to their account and view it on their device

**Comment by thecodrr** [comment]
I have added Notebook Publishing to the list of Ideas: https://notesnook.com/roadmap/. Feel free to upvote.
